src/commands/case_management.py

```diff
@@ -94,14 +100,18 @@
     + pyparsing.OneOrMore(irc_name).setResultsName("rats")
 )
 
-INJECT_PATTERN = suppress_first_word + rescue_identifier.setResultsName(
-    "subject"
-) & pyparsing.Optional(pyparsing.CaselessLiteral("cr")).setResultsName(
-    "code_red"
-) + pyparsing.Optional(
-    timer("timer")
-) + rest_of_line.setResultsName(
-    "remainder"
+INJECT_PATTERN = (
+    suppress_first_word
+    + rescue_identifier.setResultsName("subject")
+    # The following group captures in any order (&).
+    + (
+        pyparsing.Optional(pyparsing.CaselessKeyword("cr")).setResultsName("code_red")
```

Comment:
[Question] Might it be a good idea to expand this to include "code red" as an option?

Reply:
as that takes two tokens that is somewhat involved...
My attempts to add `"code red"` as a recognized term only produced more problems...

Reply:
Nvm, figured it out. Will include!

Reply:
![image](https://user-images.githubusercontent.com/3110986/93127611-0c8df500-f683-11ea-9bac-bba7749ae2ed.png)